e2e/multi-svc-app/back-end/main.go

```diff
@@ -23,17 +23,17 @@ func SimpleGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
 	w.Write([]byte("back-end"))
 }
 
-// ServiceDiscoveryGet just returns true no matter what
-func ServiceDiscoveryGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
-	log.Println("Get on ServiceDiscovery endpoint Succeeded")
+// Get just returns true no matter what
```

Comment:
another not-your-fault nit
```suggestion
// Get just returns true no matter what.
```